frontend/packages/dev-console/src/components/PodsRing.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { VictoryLabel } from 'victory';
+import * as _ from 'lodash';
+import {
+  K8sResourceKind,
+  k8sPatch,
+  K8sKind,
+  SelfSubjectAccessReviewKind,
+} from '@console/internal/module/k8s';
+import { AngleUpIcon, AngleDownIcon } from '@patternfly/react-icons';
+import { checkEditAccess } from '@console/internal/components/utils';
+import PodStatus from './topology/shapes/PodStatus';
+import { Pod } from './topology/topology-types';
+import SvgDefsProvider from './svg/SvgDefsProvider';
+import { calculateRadius } from './topology/shapes/WorkloadNode';
+
+interface PodsRingProps {
+  pods: Pod[];
+  obj: K8sResourceKind;
+  resourceKind: K8sKind;
+  path: string;
+  impersonate?: string;
+}
+
+const PodsRing: React.FC<PodsRingProps> = ({ pods, obj, resourceKind, path, impersonate }) => {
+  const [editable, setEditable] = React.useState(false);
+  const ringContainerSize = 200;
+  const size = ringContainerSize - 80;
+  const calculatedRadius = calculateRadius(size);
+  const { radius } = calculatedRadius;
+  const { podStatusInnerRadius } = calculatedRadius;
+  const { podStatusOuterRadius } = calculatedRadius;
+
+  React.useEffect(() => {
+    checkEditAccess(obj, resourceKind, impersonate)
+      .then((resp: SelfSubjectAccessReviewKind) => setEditable(resp.status.allowed))
+      .catch((error) => {
+        throw error;
+      });
+  }, [pods, obj, resourceKind, impersonate]);
+
+  const handleScaling = (operation: number) => {
+    const patch = [{ op: 'replace', path, value: _.toInteger(obj.spec.replicas) + operation }];
+    const promise: Promise<K8sResourceKind> = k8sPatch(resourceKind, obj, patch);
+    promise.catch((error) => {
+      throw error;
+    });
+  };
+
+  return (
+    <React.Fragment>
+      <svg width={ringContainerSize} height={ringContainerSize}>
+        <SvgDefsProvider>
+          <g transform="translate(0, 35)">
```

Comment:
I suspect the 35 down is for the tooltip?
We should consider using the standard console or PF tooltip instead of the tooltip in SVG so that it can float over other nodes.